src/app/modal/components/lose-changes/lose-changes.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "node_modules/govuk-frontend/govuk/all";
+
+.mat-dialog-content {
```

Comment:
I guess we do need to style the component since it is not provided by gds,
Can we please move the styling into a _modal.scss file (we make partial import so it is not done as a separate css file) that we then load into global.scss in the components section instead please?
We avoid creating framework dependencies if we had later reuse the css.

Also let's please leverage the sass variables given by gds
https://design-system.service.gov.uk/styles/colour/

If we can it would best to use relative size instead of absolute (especially for the width)